### PR TITLE
Pinterest updates

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Added FAQ's and updated some information

### CODE REVIEW

**Review by utsabc — APPROVED** [review]

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx [hunk 1]
```diff
@@ -184,4 +180,20 @@ The following table lists the custom fields mappings for `track` and `page` call
...
+| `properties.query` | `search_string` | String |
+
+## FAQ
+
+### How can I verify if my events are being sent to Pinterest Conversions API?
+
+Follow these steps to see your events in Pinterest Conversions API:
+
+1. Login to your [Pinterest ads manager](https://ads.pinterest.com/) account.
+2. Click **Ads** tab and select **Conversions** from the dropdown.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:192]:
```suggestion
2. Click the **Ads** tab and select **Conversions** from the dropdown.
```

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx [hunk 2]
```diff
@@ -184,4 +180,20 @@ The following table lists the custom fields mappings for `track` and `page` call
...
+3. Select **API for conversions** from the dropdown to see your events, as shown:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-cloud-mode.mdx:193]: should we have the image before the info block for continuity?

File: docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-device-mode.mdx [hunk 3]
```diff
@@ -169,3 +163,13 @@ rudderanalytics.page("Best Seller", {
...
 ```
 
+## FAQ
+
+### How can I verify if my events are being sent to Pinterest Tag?
+
+Follow these steps to see your events in Pinterest Tag:
+
+1. Login to your [Pinterest ads manager](https://ads.pinterest.com/) account.
+2. Click **Ads** tab and select **Conversions** from the dropdown to see your events, as shown:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/pinterest-ads/pinterest-ads-device-mode.mdx:173]:
```suggestion
2. Click the **Ads** tab and select **Conversions** from the dropdown to see your events, as shown:
```

**Review by ameypv-rudder — APPROVED** [review]
Made some minor suggestions - LGTM overall.

**Pull request merged**